### PR TITLE
Card Web Component

### PR DESCRIPTION
**Pull request by benja**
Added the Card to the Web Component collection.

Documentation shows example on how to use Card with an anchor element to ensure correct semantics for accessible use, as well as how to use the onClick handler to set selected state and perform other actions on click.

### CODE REVIEW

**Comment by benja** [comment]
Now also generates proper types when built.

File: packages/card/index.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import { FabricWebComponent } from '../utils';
+import { card as c } from '@fabric-ds/component-classes';
+import { classNames } from '@chbphone55/classnames';
+
+class FabricCard extends FabricWebComponent {
+  static get observedAttributes() {
+    return ['onlick', 'selected'];
```
Comment (digitalsadhu) [inline packages/card/index.js:7]: Tehehehehe. You see it? You see what I’m chuckling at? Best typo ever!

File: packages/card/index.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  }
+
+  attributeChangedCallback(name, _, newValue) {
+    switch (name) {
+      case 'onclick':
```
Comment (digitalsadhu) [inline packages/card/index.js:12]: Do we need to handle this ourselves? Wouldn’t we get onclick for free?

File: packages/card/index.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        break;
+      case 'selected':
+        this[name] = newValue == 'false' ? false : true;
+        break;
+      default:
+        this[name] = newValue;
+    }
+
+    this.id =
+      this.id ||
```
Comment (digitalsadhu) [inline packages/card/index.js:23]: Do we need to set an id internally?

File: packages/card/index.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+      </${this.as}>
+      `;
+
+    this.handleSetup();
+  }
+
+  handleSetup() {
+    const el = this.shadowRoot.getElementById(`${this.id}`);
+
+    if (this.onclick) {
```
Comment (digitalsadhu) [inline packages/card/index.js:96]: I wonder if we should be allowing the user to set a type instead and then defer to built in event handling  for events. 

If the user set type=button or whatever we could display a button which I think should just bubble if they define a typical onclick event on the custom element. Maybe.

**Review by digitalsadhu — COMMENTED** [review]
Some thoughts added.

File: packages/card/index.js (hunk 1 above)
Comment (benja) [reply packages/card/index.js:7]: This is not a bug, this is a feature! 😆

File: packages/card/index.js (hunk 2 above)
Comment (benja) [reply packages/card/index.js:12]: This is true, I was under the impression that it returned a string and not actually a callable function, but seems to be working without this.

File: packages/card/index.js (hunk 3 above)
Comment (benja) [reply packages/card/index.js:23]: For re-rendering purposes, yeah. I tried without with no success.

**Pull request merged**